### PR TITLE
fix(dh): message archive v2 feedback

### PR DESCRIPTION
**Pull request by ManBearTM**
fix: do not open swagger UI when launching BFF
fix: add dash between RSM and XXX in translations
fix: do not persist form state across page boundaries in message archive
fix: pagination now works again if navigating away and then back to a table that uses ApolloDataSource.
fix: subscribeToMore no longer attempts to merge with cache when result is loading (showed as errors in log)
fix: no more EmptyError when destroying components using ApolloDataSource. 
fix: deep link to calculation in calculations view now works again
fix: nextFetchPolicy now actually works with refetch in `util-apollo`
fix: prevent exception in drawer related to viewChild not being immediately available
fix: watt-paginator now no longer requires a refresh to update translations
fix: wattQueryParams directive no longer removes existing query parameters
refac: updated calculations details to use `util-apollo`

### CODE REVIEW

File: libs/dh/wholesale/feature-calculations/src/lib/details/details.component.ts [hunk 1]
```diff
@@ -81,21 +70,23 @@ import { WattModalActionsComponent, WattModalComponent } from '@energinet-datahu
...
+    nextFetchPolicy: 'cache-first',
+  });
 
-  CalculationOrchestrationState = CalculationOrchestrationState;
+  result = computed(() => this.query.data()?.calculationById);
+  type = computed(() => this.result()?.calculationType ?? 'UNKNOWN');
+  executionType = computed(() => this.result()?.executionType);
+  statusType = computed(() => this.result()?.statusType ?? 'skeleton');
+  state = computed(() => this.result()?.state ?? 'INDETERMINATE');
+  isScheduled = computed(() => this.result()?.state === State.Scheduled);
```
Comment (dzhavat) [inline libs/dh/wholesale/feature-calculations/src/lib/details/details.component.ts:89]: praise: nice change. Also quite cool that the properties will re-run/update only if the `result` Signal changes. No more worrying about excessive template re-rendering 🚀

File: libs/dh/shared/util-apollo/src/lib/query.ts [hunk 2]
```diff
@@ -200,7 +246,8 @@ export function query<TResult, TVariables extends OperationVariables>(
         .pipe(
           map(({ data }) => data),
           exists(), // The data should generally be available, but types says otherwise
-          withLatestFrom(refReplay$), // Ensure the ref (and cache) is ready,
+          withLatestFrom(refReplay$), // Ensure the ref (and cache) is ready
+          delayWhen(() => result()), // Do not attempt to merge when result is loading
           takeUntilDestroyed(destroyRef), // Stop the subscription when the component is destroyed
           takeUntil(reset$) // Or when resetting the query
         )
```
Comment (dzhavat) [inline libs/dh/shared/util-apollo/src/lib/query.ts:253]: praise: good that we have tests to cover these functions. Otherwise it's untouchable 😝

**Review by dzhavat — DISMISSED** [review]
praise: great, great work 🚀

**Review by dzhavat — APPROVED** [review]

File: libs/dh/shared/util-apollo/src/lib/query.ts (hunk 2 above)
Comment (ManBearTM) [reply libs/dh/shared/util-apollo/src/lib/query.ts:262]: Not sure if sarcastic or not, but now there are actual tests for this specific scenario 😆 
https://github.com/Energinet-DataHub/greenforce-frontend/pull/3617/commits/2d62112e3ca1dfbf39dcc186d529be67522267f5

Comment (dzhavat) [reply libs/dh/shared/util-apollo/src/lib/query.ts:256]: Wasn't sarcastic, sorry for making it ambiguous. It was a genuine comment about the tests that come along with the functions. I think they're invaluable to have, and make future changes easier. So great work again. 👍

**Review by dzhavat — APPROVED** [review]

**Pull request merged**